wemake_python_styleguide/cli/commands/explain/violation_loader.py

```diff
@@ -0,0 +1,91 @@
+"""Provides tools to extract violation info."""
+
+import importlib
+import inspect
+from collections.abc import Collection, Mapping
+from types import ModuleType
+from typing import Final, final
+
+from attrs import frozen
+
+from wemake_python_styleguide.violations.base import BaseViolation
+
+_VIOLATION_SUBMODULES: Final = (
```

Comment:
I don't like that it is a constant. We might forget about it in some time. Let's use `glob` or something.